assets/javascripts/discourse/components/region-input.js.es6

```diff
@@ -0,0 +1,288 @@
+import ComboBoxComponent from "select-kit/components/combo-box";
+import { computed } from "@ember/object";
+
+// List of regions supported by the 'holidays' gem
+// require 'holidays' ; Holidays.available_regions
+const HOLIDAY_REGIONS = [
```

Comment:
Can we have this list automatically generated by `Holidays.available_regions` minus the ones that are too specific?

Reply:
I can render this on the server with a `.js.erb` file. I will try that.

Reply:
That might not work once we migrate to ember-cli though.

Reply:
I implemented a rake task for it.